pkg/aws/eni/node.go

```diff
@@ -559,8 +559,19 @@ func (n *Node) SyncToAPIServer() (err error) {
 		updatedNode, err = n.manager.k8sAPI.UpdateStatus(node, origNode)
 		if updatedNode != nil && updatedNode.Name != "" {
 			node = updatedNode.DeepCopy()
-		}
-		if err == nil || updatedNode == nil {
+			if err == nil {
+				break
+			}
+		} else if err != nil {
+			scopedLog.WithError(err).Warningf("UpdateStatus failed: %+v", updatedNode)
```

Comment:
updateNode will be nil, why printing as part of the warning?

Reply:
It was not meant to stay at all. I've added it to test everything.